src/sentry/api/endpoints/event_ai_suggested_fix.py

```diff
@@ -322,26 +322,30 @@ def get(self, request: Request, project, event_id) -> HttpResponse | StreamingHt
         if event is None:
             raise ResourceDoesNotExist
 
-        # Check the OpenAI access policy
-        policy = get_openai_policy(
-            request.organization,
-            request.user,
-            pii_certified=request.GET.get("pii_certified") == "yes",
-        )
         policy_failure = None
-        stream = request.GET.get("stream") == "yes"
-
-        if policy == "subprocessor":
-            policy_failure = "subprocessor"
-        elif policy == "individual_consent":
-            if request.GET.get("consent") != "yes":
-                policy_failure = "individual_consent"
-        elif policy == "pii_certification_required":
-            policy_failure = "pii_certification_required"
-        elif policy == "allowed":
-            pass
+        # If the option has specifically been set to False,
+        if not bool(request.organization.get_option("sentry:ai_suggested_solution", default=False)):
+            policy_failure = "organization_consent_required"
```

Comment:
I typically would suggest handling this failure case in the UI gracefully too, but as we're sunsetting this feature soon with autofix anyways, this should be fine. It will show a generic error component.